client/components/telecom/telephony/group/number/feature/ovhPabx/dialplan/edit/telephony-group-number-feature-ovh-pabx-dialplan-edit.html

```diff
@@ -18,132 +18,120 @@ <h2 data-translate="telephony_number_feature_ovh_pabx_dialplan_edit"></h2>
-                                </flat-switch>
-                            </div>
-                            <div class="dtd text-center">
-                                <button type="button"
-                                        class="no-style text-center space-p0"
-                                        data-ng-class="{ bold: $ctrl.dialplan.anonymousRejection }"
-                                        data-ng-click="$ctrl.dialplan.anonymousRejection = true"
-                                        data-translate="telephony_number_feature_ovh_pabx_dialplan_anonymous_rejection_active">
-                                </button>
+                        <div class="d-table w-100 table-fixed">
+                            <div class="oui-checkbox">
+                                <input type="checkbox"
+                                       class="oui-checkbox__input"
+                                       id="call-accept-anonymous-checkbox"
+                                       data-ng-model="$ctrl.dialplan.anonymousRejection"
+                                       data-ng-true-value="false"
+                                       data-ng-false-value="true">
+                                <label class="oui-checkbox__label-container" for="call-accept-anonymous-checkbox">
+                                    <span class="oui-checkbox__label"
+                                          data-translate="telephony_number_feature_ovh_pabx_dialplan_anonymous_acceptation"></span>
+                                    <span class="oui-checkbox__icon">
+                                        <i class="oui-icon oui-icon-checkbox-unchecked" aria-hidden="true"></i>
+                                        <i class="oui-icon oui-icon-checkbox-checked" aria-hidden="true"></i>
+                                        <i class="oui-icon oui-icon-checkbox-checkmark" aria-hidden="true"></i>
+                                    </span>
+                                </label>
                             </div>
                         </div>
                     </div> <!-- ANONYMOUS REJECTION -->
-
                 </div>
-
             </section>
 
-            <footer class="clearfix double-choice">
-                <button type="submit"
-                        data-ng-click="$ctrl.onValidateBtnClick()"
-                        class="btn btn-primary"
-                        data-ng-disabled="($ctrl.dialplan.status !== 'DRAFT' && !$ctrl.dialplan.hasChange()) || dialplanForm.$invalid"
-                        data-translate="{{ $ctrl.dialplan.status === 'DRAFT' ? 'common_create' : 'modify' }}">
-                </button>
-
-                <button type="button"
-                        class="btn btn-default"
-                        data-translate="cancel"
-                        data-ng-click="$ctrl.onCancelBtnClick()">
-                </button>
+            <footer class="clearfix">
+                <div class="btn-group pull-right">
```

Comment:
`pull-right` => `float-right`

Reply:
This is exactly the same thing, https://bootstrapdocs.com/v3.3.5/docs/css/#helper-classes-floats
Also, pull-right is use 61 other times in 57 files and this is without components